crates/wasmparser/Cargo.toml

```diff
@@ -14,6 +14,7 @@ exclude = ["benches/*.wasm"]
 
 [dependencies]
 indexmap = { workspace = true }
+url = "2.3.1"
```

Comment:
I feel like we may need to one day add a `component-model` feature to `wasmparser` given the relative weight of the implementation internally and the added dependencies, but that's an issue for another day.